r1150375 | sedwards | 2010-07-15 22:17:52 +0200 (Thu, 15 Jul 2010) | 2 lines
Changed paths:
   M /trunk/KDE/kdebindings/python/pykde4/sip/akonadi/addressattribute.sip

Small compatibility fix for the latest (minor) version of SIP which came out a couple days ago.

Applied-Upstream: 4.6.0

--- a/python/pykde4/sip/akonadi/addressattribute.sip
+++ b/python/pykde4/sip/akonadi/addressattribute.sip
@@ -57,6 +57,10 @@ public:
     else if (dynamic_cast<Akonadi::MessageThreadingAttribute*>(sipCpp))
         sipType = sipType_Akonadi_MessageThreadingAttribute;
 %End
+private:
+//force
+Akonadi::AddressAttribute& operator = (const Akonadi::AddressAttribute&);
+//end
 };
 };
 %ModuleHeaderCode
